tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -113,6 +113,12 @@ public final String toString() {
             return j.toString();
         }
 
+        /**
+         * This converts a CheckStatus object to JSON.
```

Comment:
How about "Convert this CheckStatus to JSON"?

Reply:
Yes, that would be a clearer name.